etc/ci/performance/test_perf.sh

```diff
@@ -34,4 +34,9 @@ pip install "treeherder-client>=3.0.0"
 mkdir -p servo
 mkdir -p output
 ./git_log_to_json.sh > servo/revision.json && \
-./test_all.sh --servo
+
+if [[ "${1}" = "--submit" ]]; then
```

Comment:
You need to guard this by checking the number of arguments before looking at any of their values.